pluginconfig/commit.go

```diff
@@ -26,6 +26,7 @@ const (
 	defaultRemoteGasPriceBatchWriteFrequency  = 1 * time.Minute
 	defaultSignObservationPrefix              = "chainlink ccip 1.6 rmn observation"
 	defaultTransmissionDelayMultiplier        = 30 * time.Second
+	defaultInflightPriceCheckRetries          = 3
```

Comment:
Assuming 1s per round is this really a reasonable # of rounds for a default?

Reply:
I don't think one block is a reasonable expectation to get a tx in, especially in periods of high traffic.